ee/modules/160-multitenancy-manager/crds/doc-ru-projects.yaml

```
@@ -10,15 +10,16 @@ spec:
               properties:
                 description:
                   description: Произвольное описание назначения окружения.
-                projectTypeName:
+                templateName:
```

```suggestion
                projectTypeName:
```
The field cannot be renamed.